src/color/structuredata-colormaker.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file Colordata Colormaker
+ * @author Fred Ludlow <fred.ludlow@gmail.com>
+ * @private
+ */
+
+import { ColormakerRegistry } from '../globals'
+import Colormaker, { ColormakerScale, manageColor, StuctureColormakerParams } from './colormaker'
+import AtomProxy from '../proxy/atom-proxy'
+import BondProxy from '../proxy/bond-proxy'
+
+
+class StructuredataColormaker extends Colormaker {
+  atomData?: { [key: number]: number }
```

Comment:
same as above, expected format is `number[]`. For consistency we could also import the `ColorData` interface here and define `atomData` type as `ColorData['atomData']`